samples/javaee8-webprofile-liberty/javaee8-webprofile-liberty-app/src/main/java/io/fabric8/maven/sample/javaee/webprofile/servlet/DemoServlet.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.fabric8.maven.sample.javaee.webprofile.servlet;
+
+import java.io.IOException;
+
+import java.util.Properties;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/demo")
+public class DemoServlet extends HttpServlet {
+
+	private static final long serialVersionUID = 3745576337037604707L;
```

Comment:
Servlet is just another API under the Java EE umbrella, I think it's not a problem to have it here.

Reply:
I'm talking about serialVerionUID variable

Reply:
Because the `HttpServlet` class is serializable, the `serialVersionUID` constant must be declared. My Eclipse marks the class with a warning if the constant isn't defined there. I've found a good explanation on [SO](https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it).